main/end-to-end-tests/checkConnection.js

```diff
@@ -0,0 +1,11 @@
+// This file is used for checking whether an IP Address is reachable on a particular port
+// eslint-disable-next-line import/no-extraneous-dependencies
+const tcpp = require('tcp-ping');
+
+const [ipAddress, port] = process.argv.slice(2);
+(async () => {
+  const tcpPingPromise = (ipAddr, portForIp) =>
+    new Promise((resolve, reject) => tcpp.probe(ipAddr, portForIp, (err, data) => (err ? reject(err) : resolve(data))));
+  const response = await tcpPingPromise(ipAddress, port);
+  console.log(response);
```

Comment:
This is logging ping response without context. Let's remove the `console.log` statement, or add info to make the content readable.

Reply:
I see stdout is being used for test verification. Can we instead have this function return a boolean, and use that for `expect` checks?